src/qforte/computer.cc

```diff
@@ -874,7 +874,7 @@ std::complex<double> Computer::direct_pauli_circ_exp_val(const Circuit& qc) {
     std::vector<int> y_idxs;
     std::vector<int> z_idxs;
 
-    for (const Gate& gate : qc.gates()) {
+    for (const auto& gate : qc.gates()) {
```

Comment:
Out of curiosity,  why was this changed to auto?

Reply:
[Using C++ auto is best practice by default.](https://herbsutter.com/2013/08/12/gotw-94-solution-aaa-style-almost-always-auto/)